autogpt/commands/execute_code.py

```diff
@@ -115,6 +115,11 @@ def execute_shell(command_line: str) -> str:
     Returns:
         str: The output of the command
     """
+    
+    # Get disallowed commands from the environment variable, check if any are in the command line input.
+    disallowed_commands = [cmd.strip() for cmd in os.getenv('DISALLOWED_COMMANDS').split(',')]
+    if any(command in command_line for command in disallowed_commands):
+        return ("Error: Disallowed command detected. Use another command to fulfill task")
```

Comment:
This should not be pulled from the OS.env. It should be passed in as appropriate as this clutters the global state

Reply:
You're right of course, I am thinking to add a small function outside the execute_shell function to load the disallowed commands so they can be used as arguments from there

So the execute_shell function can accept disallowed commands from elsewhere too, correct ?